pkg/csi/service/wcp/controller.go

```diff
@@ -164,6 +176,13 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 		log.Errorf("failed to watch on path: %q. err=%v", cfgDirPath, err)
 		return err
 	}
+	caFileDirPath := filepath.Dir(cnsconfig.SupervisorCAFilePath)
+	log.Infof("Adding watch on path: %q", caFileDirPath)
+	err = watcher.Add(caFileDirPath)
```

Comment:
we are adding multiple paths to single watcher. Where do we check and how do we identify fsnotify event is for the intended path in our logic.

Reply:
Line: 151 handles this